src/Layers/xrRenderPC_R1/FStaticRender.cpp

```diff
@@ -91,6 +91,10 @@ void CRender::create()
     o.no_detail_textures = !ps_r2_ls_flags.test(R1FLAG_DETAIL_TEXTURES);
     c_ldynamic_props = "L_dynamic_props";
 
+    o.managed_tex_disabled = (strstr(Core.Params, "-nomanagedtex")) ? TRUE : psDeviceFlags.test(rsDisableManagedTex);
```

Comment:
Let's change CLI option name to `-no_staging` to be consistent with R3+

Reply:
I can, but "staging" word is mostly from DX10\DX11 (D3D11_USAGE_**STAGING**)
For DX9 is D3DPOOL_**MANAGED**

Currently "-nomanagedtex" works for all renders, "-no_staging" only in DX10\11

Reply:
The "staging" is just a common word for the resources transferring technique. Managed resources behave the same as staged but this is hidden by driver logic. The meaning of this option is the same for all renderers

Reply:
Yeah, because of the different lifetime of RAM textures (that the point of PR), just realized that `-nostaging` is confusing naming for DX9, indeed.
Since the option can be described as "keep textures in VRAM only" probably it's better to name it like `-noramtex`. And rename DX10 `-nostaging` to the same name accordingly.